lametro/models.py

```diff
@@ -458,6 +459,31 @@ def upcoming_committee_meetings(cls):
         return meetings
 
 
+    @property
+    def event_minutes(self):
```

Comment:
is it true that this only ever returns values for board meetings (and not subcommittee meetings, etc.)? if so, could we:

1. check whether the event is a board meeting up top, and return none if it isn't, to avoid making queries when we don't need to?
2. rename this to `board_meeting_minutes`, to make it a bit more explicit?

Reply:
oh, i see we check whether it's a board meeting in the template, so we won't save queries for this use case, but it would be nice to bundle that all into this property, so it's clear if we want to make use of it later.

Reply:
Agreed! (And what I originally had: https://github.com/datamade/la-metro-councilmatic/pull/424/commits/6884be4d4371ce28ef28cb3635dd1a189320bf72#diff-154e85485bdcc3826d208aa9dbc8d8e3R471)